packages/medusa/src/api/routes/admin/payment-collections/delete-payment-collection.ts

```diff
@@ -0,0 +1,13 @@
+import { PaymentCollectionService } from "../../../../services"
+
+export default async (req, res) => {
+  const { id } = req.params
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const payment_collection = await paymentCollectionService.delete(id)
```

Comment:
todo: Remove const assignment
```suggestion
  await paymentCollectionService.delete(id)
```